elements/a.html

```diff
@@ -1,30 +1,56 @@
 <article class="article">
   <header class="article-header">
-    <h2>
+        <style type="text/css">
+        button {
+            border: none;
+            cursor: pointer;
+            outline: none;
+            position: relative;
+        }
+
+        button.play-button {
+            background: url('/button/play.png') no-repeat;
+
+            width: 2em;
+            height: 1.8em;
+            background-size: 20px 20px;
+            margin-left: 3px;
+        }
+
+        button.play-button.pause {
+            background: url('/button/pause.png') no-repeat;
+            background-size: 20px 20px;
+
+        }
+        </style>
```

Comment:
Let's take out all this style stuff from the HTML and put it in a css file.  Based on the size of this PR and the amount of merge conflicts I think it would be better to merge this and create a new PR for pulling out the in-line styling, but it should be a priority to do that work after this is merged.